src/HotChocolate/Core/src/Types.Scalars/HexType.cs

```diff
@@ -4,30 +4,30 @@
 namespace HotChocolate.Types.Scalars
 {
     /// <summary>
-    /// The HexColorCode scalar type represents a valid hex color code.
+    /// The Hex scalar type represents a valid hex color code.
     /// </summary>
-    public class HexColorCodeType : StringType
+    public class HexType : StringType
```

Comment:
I think this is to generic. 
`Hex` could also be something other than a color code i think